### ISSUE TITLE
Build errors: error: base class 'mesh_t' has a flexible array member, etc.

### ISSUE THREAD

**Issue by yurivict** [sadko4u/lsp-plugins#246]
```
In file included from wrap/jack.cpp:43:
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/types.h:150:39: error: base class 'mesh_t' has a flexible array member
        typedef struct mesh_t: public plug::mesh_t
                                      ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/types.h:169:23: error: no member named 'nState' in 'lsp::jack::mesh_t'
                mesh->nState        = plug::M_EMPTY;
                ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/types.h:170:23: error: no member named 'nBuffers' in 'lsp::jack::mesh_t'
                mesh->nBuffers      = 0;
                ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/types.h:171:23: error: no member named 'nItems' in 'lsp::jack::mesh_t'
                mesh->nItems        = 0;
                ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/types.h:175:27: error: no member named 'pvData' in 'lsp::jack::mesh_t'
                    mesh->pvData[i]    = reinterpret_cast<float *>(ptr);
                    ~~~~  ^
In file included from wrap/jack.cpp:45:
In file included from /disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_wrapper.h:30:
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:188:51: error: no member named 'containsData' in 'lsp::jack::mesh_t'
                    if ((mesh == NULL) || (!mesh->containsData()))
                                            ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:192:48: error: no member named 'nBuffers' in 'lsp::jack::mesh_t'
                    for (size_t i=0; i < mesh->nBuffers; ++i)
                                         ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:193:52: error: no member named 'pvData' in 'lsp::jack::mesh_t'
                        dsp::copy_saturated(pMesh->pvData[i], mesh->pvData[i], mesh->nItems);
                                            ~~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:193:69: error: no member named 'pvData' in 'lsp::jack::mesh_t'
                        dsp::copy_saturated(pMesh->pvData[i], mesh->pvData[i], mesh->nItems);
                                                              ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:193:86: error: no member named 'nItems' in 'lsp::jack::mesh_t'
                        dsp::copy_saturated(pMesh->pvData[i], mesh->pvData[i], mesh->nItems);
                                                                               ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:194:28: error: no member named 'data' in 'lsp::jack::mesh_t'
                    pMesh->data(mesh->nBuffers, mesh->nItems);
                    ~~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:194:39: error: no member named 'nBuffers' in 'lsp::jack::mesh_t'
                    pMesh->data(mesh->nBuffers, mesh->nItems);
                                ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:194:55: error: no member named 'nItems' in 'lsp::jack::mesh_t'
                    pMesh->data(mesh->nBuffers, mesh->nItems);
                                                ~~~~  ^
/disk-samsung/freebsd-ports/audio/lsp-plugins-lv2/work/lsp-plugins/modules/lsp-plugin-fw/include/lsp-plug.in/plug-fw/wrap/jack/ui_ports.h:197:27: error: no member named 'cleanup' in 'lsp::jack::mesh_t'
                    mesh->cleanup();
                    ~~~~  ^
```

Version: 1.2.0
clang-13
FreeBSD 13

**Comment by sadko4u** [comment]
Thanks. Will provide the patch soon.

**Comment by sadko4u** [comment]
This patch should fix the problem:
https://github.com/lsp-plugins/lsp-plugin-fw/commit/b2d31f757611b6fb8fc16c8a78465b11bc392526

**Comment by sadko4u** [comment]
Please don't close tickets with solutions.
I will close them when the next release will be issued.

**Comment by sadko4u** [comment]
Available in 1.2.1.
